acbuild/acbuild.go

```diff
@@ -131,6 +131,16 @@ func runWrapper(cf func(cmd *cobra.Command, args []string) (exit int)) func(cmd
 			return
 		}
 
+		if len(aciToModify) == 0 {
```

Comment:
Turns out the first thing I do in this function is `if aciToModify == ""`, so this check is redundant. I'll remove it.